Source/gg2/Scripts/Events/doEventPlayerDeath.gml

```diff
@@ -178,7 +178,7 @@ with(victim.object) {
             break;
         }
         playsound(x,y,Gibbing);
-    } else {
+    } else if (!hasClassReward(victim, 'Tombstone')) {

```

Comment:
It was, because there was supposed to be a tombstone-specific gong or something. But that didn't materialise, so maybe the suppression should be removed.